streams/src/test/java/org/apache/kafka/streams/integration/GlobalKTableEOSIntegrationTest.java

```diff
@@ -432,13 +424,14 @@ public void onRestoreEnd(final TopicPartition topicPartition,
         );
     }
 
-    @Test
-    public void shouldNotRestoreAbortedMessages() throws Exception {
+    @ParameterizedTest
+    @MethodSource("data")
```

Comment:
`@ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE, StreamsConfig.EXACTLY_ONCE_V2})`

Reply:
I've addressed all the comments, thanks for your patience 🙇